src/components/UserPortal/ChatRoom/ChatRoom.tsx

```
@@ -192,18 +196,24 @@ const MessageImageBase: React.FC<IMessageImageProps> = ({
   }, [media, organizationId, getFileFromMinio]);
 
   if (imageState.loading) {
-    return <div className={styles.messageAttachment}>Loading image...</div>;
+    return <div className={styles.messageAttachment}>{t('loadingImage')}</div>;
+    return <div className={styles.messageAttachment}>{t('loadingImage')}</div>;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)